components/popover/index.js

```diff
@@ -100,27 +96,22 @@ class Popover extends Component {
 			return;
 		}
 
-		const { content } = this.nodes;
-		if ( ! content ) {
+		if ( ! this.contentNode.current ) {
```

Comment:
Related to [earlier comment](https://github.com/WordPress/gutenberg/pull/6799/files#r206211471), we shouldn't need to check this here. If the component causes `Popover#focus` to be called during its `componentDidMount`, we should be able to assume that the `ref.current` is assigned.